destination/destination.go

```diff
@@ -0,0 +1,62 @@
+package destination
```

Comment:
This is duplicated in [steps-lookup-xcode-simulator](https://github.com/bitrise-steplib/steps-lookup-xcode-simulator/blob/master/destination/destination.go). I plan to migrate it to [go-xcode](https://github.com/bitrise-io/go-xcode) and write tests for it after this PR is merged to avoid blowing up the scope too much.